docs/decisions/0010-include-eslint-plugin-vuejs-accessibility.md

```diff
@@ -0,0 +1,88 @@
+# 0010 Use eslint-plugin-vuejs-accessibility 
+
+Date: 2024-10-13 \
+Status: Proposed <!-- Proposed | Accepted | Rejected | Superceded -->
+
+## Context and Problem Statement
+
+Accessibility is a crucial aspect of web development, and it is essential to ensure that our Vue.js applications are accessible to all users.   [eslint-plugin-vuejs-accessibility](https://github.com/vue-a11y/eslint-plugin-vuejs-accessibility) is a static checking tool that helps us identify accessibility issues in our Vue.js code.
+
+## Decision Drivers <!-- optional -->
+
+- We want to ensure that our Vue.js applications are accessible to all users.
+<!-- numbers of drivers can vary -->
+
+## Considered Options
+
+- Do not include an eslint-plugin for accessibility
+- [eslint-plugin-vuejs-accessibility](https://www.npmjs.com/package/eslint-plugin-vuejs-accessibility) 
+- [eslint-plugin-vue-a11y](https://www.npmjs.com/package/eslint-plugin-vue-a11y) 
+<!-- numbers of options can vary -->
+
+## Decision Outcome
+
+TBD
+
+<!-- Chosen option: "[option 1]", because [justification. e.g., only option, which meets k.o. criterion decision driver | which resolves force force | … | comes out best (see below)]. -->
+
+### Positive Consequences <!-- optional -->
+
+- Catching the accessibility issues in our Vue.js applications before they are deployed to production.
+- Helping developers to write more accessible Vue.js applications.
+- Reducing future remediation costs.
+
+### Negative Consequences <!-- optional -->
+
+- Some devs may not know how to fix accessibility issues in Vue.js applications, and this may cause some friction in development or result in a number of rules being disabled.
```

Comment:
I think it's likely that we could encounter this from time to time, although IMO that's certainly not a deal-breaker. That said, I'm wondering if we might be able to mitigate issues for newcomers by providing documentation – do you think there are any useful resource(s) that would be worth linking to from our `docs/` directory, as a starting point?

Reply:
If this was adopted, I would write up some introductory documentation.   The package has some documentation, but their solution isn't always the most optimal for accessibility.   

https://vue-a11y.github.io/eslint-plugin-vuejs-accessibility/rules/click-events-have-key-events.html


---------------
There is a ton of resources - will need to look for something that would be not overwhelming.
* [Deque has some good resources](https://media.dequeuniversity.com/public/en/docs/deque_web_accessibility_checklist.pdf)

I like your idea of creating issues if items need to be temporarily disabled, and we can use a common tag, so devs can see examples from the code.

------------

Reply:
That would be excellent – thank you!